package.json

```diff
@@ -48,8 +48,19 @@
     "turbo-ignore": "1.10.3",
     "typescript": "5.0.4"
   },
-  "packageManager": "npm@9.5.0",
+  "pnpm": {
+    "overrides": {
+      "@opentelemetry/api": "1.4.1",
+      "@opentelemetry/core": "1.13.0",
+      "@opentelemetry/exporter-trace-otlp-grpc": "0.39.1",
+      "@opentelemetry/resources": "1.13.0",
+      "@opentelemetry/sdk-trace-base": "1.13.0",
+      "@opentelemetry/sdk-trace-node": "1.13.0",
+      "@opentelemetry/semantic-conventions": "1.13.0"
```

Comment:
was getting the same error as this issue when building with pnpm: https://github.com/contentlayerdev/contentlayer/issues/506

it can be a temporary solution until the main issue with the dependencies are resolved (assuming the version of the contentlayer is incremented on this repo as well)